sopel/bot.py

```diff
@@ -587,32 +587,47 @@ def call_rule(
         sopel: 'SopelWrapper',
         trigger: Trigger,
     ) -> None:
+        nick = trigger.nick
+        sender = trigger.sender
```

Comment:
As an aside, I've come to dislike `trigger.sender` as an attribute name, because it sounds like that should be the `nick` who triggered the command. While you're creating this variable and updating usage of the attribute below, can this be called `channel` instead?

Reply:
The thing is: it could be a nick or a channel. So on one hand I agree with you, I don't like "sender", but on the other `channel` is not always accurate either.

Something like `origin` or `source`? `maybe_channel_maybe_nick`? :grin:

Reply:
`context`?

Reply:
Done!